test/index.js

```diff
@@ -183,3 +183,171 @@ describe('metalsmith-serve with custom indexFile', function(){
 
 });
 
+
+// not_found file serving and redirects
+describe('metalsmith-serve', function() {
+
+  var metalsmith;
+  var servePlugin;
+
+  before(function(done) {
+    metalsmith = Metalsmith("test/fixtures/site");
+
+    servePlugin = serve({
+      verbose: false,
+      "port": port,
+      "not_found": "/404.html",
```

Comment:
I would've kept the redirect separate like you had them. node-static generates these errors, some of which could have custom handling: 400, 403, 404 (duh!), 500.